cypress/utils/commands.js

```diff
@@ -30,10 +30,49 @@ Cypress.Commands.add('getElementsByTestIds', (testIds, options = {}) => {
   return cy.get(selectors.join(','), options);
 });
 
-Cypress.Commands.add('getElementByDataTestId', (testId) => {
-  return cy.get(`[data-testid="${testId}"]`);
+/**
```

Comment:
it doesn't appear these JSDocs help with IDEs, at least on my machine. Since these aren't regular functions, I don't think JSDocs will help us here. Could we remove them and instead add the typings to here? https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/cypress/utils/index.d.ts